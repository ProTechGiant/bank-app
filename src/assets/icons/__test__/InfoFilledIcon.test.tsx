import { create } from "react-test-renderer";

import { InfoFilledIcon } from "../InfoFilledIcon";

describe("icon/InfoFilledIcon", () => {
  it("renders a InfoFilledIcon", () => {
    const result = create(<InfoFilledIcon />);

    expect(result).toMatchSnapshot();
  });
});
