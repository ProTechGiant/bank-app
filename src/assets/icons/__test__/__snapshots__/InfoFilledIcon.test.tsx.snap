// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icon/InfoFilledIcon renders a InfoFilledIcon 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="20"
  bbWidth="16"
  fill="none"
  focusable={false}
  height={20}
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 20,
        "width": 16,
      },
    ]
  }
  vbHeight={20}
  vbWidth={16}
  width={16}
  xmlns="http://www.w3.org/2000/svg"
>
  <RNSVGGroup
    fill={null}
    propList={
      [
        "fill",
      ]
    }
  >
    <RNSVGPath
      d="M8 0L0 3V9.09C0 14.14 3.41 18.85 8 20C12.59 18.85 16 14.14 16 9.09V3L8 0ZM9 14H7V12H9V14ZM9 10H7V5H9V10Z"
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
        ]
      }
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;
